src/nyc_trees/apps/event/views.py

```diff
@@ -429,3 +445,26 @@ def increase_rsvp_limit(request, event_slug):
     return {
         'max_attendees': event.max_attendees
     }
+
+
+@transaction.atomic
+def event_email_unsubscribe(request, event_slug, token):
```

Comment:
Valid concern, but this is by design. If we check against the current user session, that assumes the user must login first, which defeats the purpose of having a one-click link to unsubscribe. It's also pretty convenient during testing. Hope that makes sense.